packages/emotion/package.json

```diff
@@ -33,16 +33,14 @@
     "@instructure/ui-react-utils": "8.24.2",
     "@instructure/ui-themes": "8.24.2",
     "@instructure/ui-utils": "8.24.2",
-    "emotion-theming": "^11",
     "hoist-non-react-statics": "^3.3.2",
     "lodash": "^4",
     "prop-types": "^15"
   },
   "devDependencies": {
     "@instructure/ui-babel-preset": "8.24.2",
     "@instructure/ui-test-utils": "8.24.2",
-    "@types/lodash": "^4.14.171",
-    "cpy-cli": "^3.1.1"
```

Comment:
According to [`depcheck`](https://github.com/depcheck/depcheck) these dependencies were not used.